llumnix/llumlet/local_migration_scheduler.py

```diff
@@ -20,14 +20,17 @@ def __init__(self, request_migration_policy: str, backend_engine: BackendInterfa
         self.request_migration_policy = request_migration_policy
         self.backend_engine = backend_engine
 
-    def get_migrate_out_request(self) -> Optional[MigratingRequest]:
+    def get_migrate_out_request(self, num_requests) -> Optional[MigratingRequest]:
         # TODO(s5u13b): remove the if-else codes
         migrate_out_request: MigratingRequest = None
-        if self.request_migration_policy == 'LCFS':
-            migrate_out_request = self.backend_engine.get_last_running_request()
-        elif self.request_migration_policy in ['SJF', 'LJF']:
-            if self.request_migration_policy == 'LJF':
-                migrate_out_request = self.backend_engine.get_longest_running_request()
-            elif self.request_migration_policy == 'SJF':
-                migrate_out_request = self.backend_engine.get_shortest_running_request()
+        if num_requests == -1:
```

Comment:
if num_requests == -1, return pre-migration requests
this looks confusing. I'd prefer:
if you have any pre-migration requests, then just return one of them (this means pre-migration always have higher prioirity than other migration policy)
And this method can just return one request. Use a loop outside to migrate multiple requests